src/codewhisperer/service/securityScanHandler.ts

```diff
@@ -201,3 +207,45 @@ export async function uploadArtifactToS3(fileName: string, resp: CreateUploadUrl
     })
     getLogger().debug(`StatusCode: ${response.statusCode}`)
 }
+
+export async function patchVulnerableCode({ filePath, issues: [issue] }: AggregatedCodeScanIssue) {
```

Comment:
Why is this the function signature? From the caller it looks like you can apply multiple patches but only the first one is being taken?